src/switch.c

```diff
@@ -17,6 +17,7 @@
 #include <sys/queue.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <netinet/ip.h>
```

Comment:
So this include is the one that defines IP_MAXPACKET. If we were to use that include here we would need to have it after the #else, since that include doesn't exist on windows.
```
#ifdef _WIN32
	#include <winsock2.h>
	#include <ws2tcpip.h>
#else
	#include <sys/types.h>
```
But since IP_MAXPACKET is defined to 65535 and that the ARP packet will be quite small, I guess we can simply define GARP_MAXPACKET to 1500 and use that instead of IP_MAXPACKET. That will allow us not to bother about which OS needs which includes and also to have a smaller buffer size.